src/backend/gporca/libnaucrates/src/parser/CParseHandlerAgg.cpp

```diff
@@ -167,11 +168,15 @@ CParseHandlerAgg::EndElement(const XMLCh *const,  // element_uri,
 	m_dxl_node = GPOS_NEW(m_mp) CDXLNode(m_mp, m_dxl_op);
 
 	// set physical properties
+	GPOS_ASSERT(NULL != prop_parse_handler);
 	CParseHandlerUtils::SetProperties(m_dxl_node, prop_parse_handler);
 
 	// add children
+	GPOS_ASSERT(NULL != proj_list_parse_handler);
 	AddChildFromParseHandler(proj_list_parse_handler);
+	GPOS_ASSERT(NULL != filter_parse_handler);
 	AddChildFromParseHandler(filter_parse_handler);
+	GPOS_ASSERT(NULL != child_parse_handler);
 	AddChildFromParseHandler(child_parse_handler);
```

Comment:
Are these changes really necessary? The original task only talks about proj_list_parse_handler.